src/rpc/rawtransaction_util.cpp

```diff
@@ -312,7 +312,8 @@ void SignTransaction(CMutableTransaction& mtx, const SigningProvider* keystore,
     // Script verification errors
     std::map<int, bilingual_str> input_errors;
 
-    bool complete = SignTransaction(mtx, keystore, coins, *nHashType, input_errors);
+    SignOptions options{.sighash_type = *nHashType};
+    bool complete = SignTransaction(mtx, keystore, coins, options, input_errors);
```

Comment:
Nit: Maybe `options` could be inlined here?

Reply:
@optout21 see https://github.com/bitcoin/bitcoin/pull/32876#issuecomment-3036784505